tests/integration/converters/__init__.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+
+import numpy as np
+import tiledb
+
+
+@dataclass(frozen=True)
+class CMU_1_SMALL_REGION:
+    _dom = {
+        0: ((0, 2219), (0, 2966)),
+        1: ((0, 386), (0, 462)),
+        2: ((0, 1279), (0, 430)),
+    }
+    _attr_dtype = [("f0", "u1"), ("f1", "u1"), ("f2", "u1")]
+    _tile = {0: (1024, 1024), 1: (387, 463), 2: (1024, 431)}
```

Comment:
Why to sacrifice the self_explanatory nature of this naming and zip values so we can save one loc?

Reply:
Because it removes the implicit assumption `len(_dom) == len(_tile)` and the hardcoded `for elem_id in range(0, 3)`. Adding (or removing) one schema now requires updating 3 lines instead of one.

Reply:
Isn't there a 1-1 relation between domain and tile? Is it an assumption do we have a case? Anyway I will proceed to the change